stored_requests/config/config.go

```diff
@@ -107,7 +107,7 @@ func CreateStoredRequests(cfg *config.StoredRequests, metricsEngine metrics.Metr
 //
 // As a side-effect, it will add some endpoints to the router if the config calls for it.
 // In the future we should look for ways to simplify this so that it's not doing two things.
-func NewStoredRequests(cfg *config.Configuration, metricsEngine metrics.MetricsEngine, client *http.Client, router *httprouter.Router) (db *sql.DB, shutdown func(), fetcher stored_requests.Fetcher, ampFetcher stored_requests.Fetcher, accountsFetcher stored_requests.AccountFetcher, categoriesFetcher stored_requests.CategoryFetcher, videoFetcher stored_requests.Fetcher) {
+func NewStoredRequests(cfg *config.Configuration, metricsEngine metrics.MetricsEngine, client *http.Client, router *httprouter.Router) (shutdown func(), fetcher stored_requests.Fetcher, ampFetcher stored_requests.Fetcher, accountsFetcher stored_requests.AccountFetcher, categoriesFetcher stored_requests.CategoryFetcher, videoFetcher stored_requests.Fetcher) {
```

Comment:
Can we remove DB connection entry that is no longer returned by this function from the comment above?
```
 96   // NewStoredRequests returns five things:
 97   //
 98 - // 1. A DB connection, if one was created. This may be nil.
 99   // 2. A function which should be called on shutdown for graceful cleanups.
100   // 3. A Fetcher which can be used to get Stored Requests for /openrtb2/auction
101   // 4. A Fetcher which can be used to get Stored Requests for /openrtb2/amp
102   // 5. A Fetcher which can be used to get Category Mapping data
103   // 6. A Fetcher which can be used to get Stored Requests for /openrtb2/video
104   //
105   // If any errors occur, the program will exit with an error message.
106   // It probably means you have a bad config or networking issue.
107   //
108   // As a side-effect, it will add some endpoints to the router if the config calls for it.
109   // In the future we should look for ways to simplify this so that it's not doing two things.
110   func NewStoredRequests(cfg *config.Configuration, metricsEngine metrics.MetricsEngine, client *http.Client, router *httprouter.Router) (shutdown func(), fetcher stored_requests.Fetcher, ampFetcher store
```

Reply:
Yes. I'll update the comment. Good catch.